.github/workflows/publish.yaml

```diff
@@ -0,0 +1,36 @@
+name: Publish package to the Maven Central Repository
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to publish. For example "0.0.1"'
+        required: true
+        default: '0.0.0'
+  release:
+    types: [ published ]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'corretto'
+          java-version: '11'
```

Comment:
i thought we upgraded to java 18, i might be wrong tho 🤔

Reply:
Should be 17 LTS - 18 is not entering LTS. @jiyoontbd please let me know if this isn't the case across the board for Java-based runtimes at TBD

Reply:
Depends on whether or not Cash is planning to use our SDKs, they have services still on 11 IIRC.